### PR TITLE
Clearly differentiate between annotation example for "Parameters" & "Return Value"

### PR DESCRIPTION
**Pull request by bashyio**
Since the documentation is targeted at TypeScript beginners, I think examples shouldn't leave things to "assumption" and be as clear as possible.

The preceding statement states **"You can use interfaces to annotate parameters and return values to functions:"**
However, the following examples are listed in the **opposite order** and do not **specify whether you're annotating parameter or return value**.

The minor edit easily clarifies this.

### CODE REVIEW

**Review by orta — APPROVED** [review]
I think this is a pretty good improvement, thanks

**Comment by bashyio** [comment]
@orta This pull request has been awaiting workflow approval for 27 days (out of 30 before it gets deleted) since I'm a first time contributor to this repo. Can you please approve the workflow run from your end? Thank you

**Comment by orta** [comment]
I'm afraid that I don't have write access to do that

**Comment by jakebailey** [comment]
I'm opting to take #2368 instead, which just swaps the order to match the previous paragraph.

**Pull request closed without merging**
